debian/control

```diff
@@ -10,7 +10,7 @@ Build-Depends: debhelper (>= 10),
                libopenmpi-dev,
                libssl-dev,
                libyaml-dev,
-               libmercury-dev (= 2.0.0~rc1-1),
+               libmercury-dev (= 2.0.0~rc3-1),
```

Comment:
Could use >= here, per the general review comment:
```suggestion
               libmercury-dev (>= 2.0.0~rc3-1),
```

Reply:
done